templates/kpi/config.es6.j2

```diff
@@ -11,3 +11,5 @@ export function getZebraLoginUrl() {
 const SENTY_DSN = "{{ kpi_raven_js_dsn }}";
 
 export const WEB_PAGE_TITLE = "{{ web_page_title }}";
+
+export const DISABLE_LIBRARY_TOGGLE = "{{ kpi_disable_library_toggle }}"
```

Comment:
I don't think it should have the quotes because the value is boolean

Reply:
Updated in latest commit